MAUI/ImageEditor/events.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: Events in .NET MAUI ImageEditor control | Syncfusion
```

Comment:
use Image Editor instead ImageEditor in all places

Reply:
Changed in all places